test/carts/postCartPaymentIT.js

```diff
@@ -0,0 +1,168 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+'use strict';
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const HttpStatus = require('http-status-codes');
+const setup = require('../lib/setupIT.js').setup;
+const requiredFields = require('../lib/requiredFields');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+
+describe('Magento postCartPayment', function () {
+
+    describe('Integration tests', function () {
+
+        // Get environment
+        let env = setup();
+
+        // Increase test timeout
+        this.slow(env.slow);
+        this.timeout(env.timeout);
+
+        let cartId;
+        let cartEntryId;
+        const productVariantId = 'eqbisumas-10';
+        let ccifPayment = {
+            token: '1234',
+            method: 'checkmo',
```

Comment:
Shouldn't we rather use `methodId` instead of `method` since this is deprecated?